openedx/features/badging/views.py

```diff
@@ -27,12 +32,37 @@ def trophycase(request):
 
     trophycase_dict = populate_trophycase(user, enrolled_courses_data, earned_user_badges)
 
+    ## Get list of dictionary keys in CourseKey Format
+    course_key_list = [CourseKey.from_string(unicode(course)) for course in trophycase_dict.keys()]
```

Comment:
Can you please update course to `course_key` or `course_id` to make it more appropriate

Reply:
Done